docs/Design/communication-adapter-interface.md

```diff
@@ -43,21 +46,51 @@ This port carries a status of `Fw::Success::SUCCESS` or `Fw::Success::FAILURE` t
 
 > Note: it is critical to obey the protocol as described in the protocol section below.
 
+## Communication Queue Protocol
+
+`Svc::ComQueue` queues messages until the communication adapter is ready to receive these messages. For each
+Fw::Success::SUCCESS message received, `Svc::ComQueue` will emit one message. `Svc::ComQueue` will not emit messages
+at any other time. This implies several things:
+
+1. An initial `Fw::Success::SUCCESS` message must be sent to `Svc::ComQueue` to initiate data flow
+2. A `Fw::Success::SUCCESS` must be eventually received in response to each message for data to continue to flow
+
+These implications are discussed in more depth in the appropriate protocol sections.
+
+## Framer Status Protocol
+
+Framing typically happens between `Svc::ComQueue` and a communications adapter. The `Svc::Framer` component implements
+this protocol in order to interoperate with these other components. The action taken by this protocol is dependent on
+the number of framed messages (frames) sent to the communication adapter in response to each message received from
+`Svc::ComQueue`.
+
+| Produced Frames | Action                                 | Rationale                                                |
+|-----------------|----------------------------------------|----------------------------------------------------------|
+| 0               | Send one `Fw::Success::SUCCESS` status | Data must continue to flow to produce a frame            |
+| 1               | Pass through received statuses         | Frame produced and communication adapter produces status |
+
+When a frame is produced the communication adapter is responsible for determining the status of the message and its
+statuses must be passed through. When no frame is produced, the communication adapter does not receive a frame and is
+incapable of producing a status. `Svc::Framer` must act appropriately in this case.
+
+> Note: `Svc::Framer` must also pass through the initiation message from the communication adapter to start data flow.
+
 ## Communication Adapter Protocol
 
-The `comStatus` port must obey a specific protocol to indicate to F´ applications the status of outgoing transmissions.
-A communication status is one of two possibilities:
+The communication adapter must obey a specific protocol to indicate to F´ application components the status of outgoing
+transmissions. This is done with the `comStatus` port. A communication status is one of two possibilities:
 
 | Status               | Description                                                                       |
 |----------------------|-----------------------------------------------------------------------------------|
 | Fw::Success::SUCCESS | *Communication adapter* transmission succeeded*  and is ready for more data.      |
```

Comment:
I think there is a stray asterisk after "succeeded."

Reply:
Fixed!